README.md

```diff
@@ -1,69 +1,89 @@
-​
+You can set the entire string in the secret value as the environment variable value, or if the string is JSON, you can parse the JSON to set individual environment variables for each JSON key\-value pair. If the secret value is a binary, the action converts it to a string.
 
 To view the environment variables created from your secrets, turn on debug logging. For more information, see [Enabling debug logging](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging) in the *GitHub Docs*.
-​
-​
-### Prerequisites
-​
-To use this action, you first need to configure AWS credentials and set the AWS Region in your GitHub environment by using the `configure-aws-credentials` step. Follow the instructions in [Configure AWS Credentials Action For GitHub Actions](https://github.com/aws-actions/configure-aws-credentials) to **Assume role directly using GitHub OIDC provider**. This allows you to use short-lived credentials and avoid storing additional access keys outside of Secrets Manager.
-​
+
+To use the environment variables created from your secrets, see [Environment variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables) in the *GitHub Docs*.
+
+## Prerequisites<a name="retrieving-secrets_github_prereq"></a>
+
+To use this action, you first need to configure AWS credentials and set the AWS Region in your GitHub environment by using the `configure-aws-credentials` step. Follow the instructions in [Configure AWS Credentials Action For GitHub Actions](https://github.com/aws-actions/configure-aws-credentials) to **Assume role directly using GitHub OIDC provider**. This allows you to use short\-lived credentials and avoid storing additional access keys outside of Secrets Manager.
+
 The IAM role the action assumes must have the following permissions:
-+ `GetSecretValue` on the secrets you want to retrieve
-+ `ListSecrets` on all secrets
-+ (Optional) `Decrypt` on the KMS key if the secrets are encrypted with a customer managed key.
-​
-For more information, see [Authentication and access control for AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/auth-and-access.html).
-​
-### Usage
-​
++ `GetSecretValue` on the secrets you want to retrieve.
++ `ListSecrets` on all secrets.
++ \(Optional\) `Decrypt` on the KMS key if the secrets are encrypted with a customer managed key.
+
+For more information, see [Authentication and access control for AWS Secrets Manager](auth-and-access.md).
+
+## Usage<a name="retrieving-secrets_github_usage"></a>
+
 To use the action, add a step to your workflow that uses the following syntax.
-​
+
 ```
 - name: Step name
   uses: aws-actions/aws-secretsmanager-get-secrets@v1
   with:
-    secret-ids: |
```

Comment:
The `|` should remain here. Checkout the 
- https://github.com/aws-actions/aws-secretsmanager-get-secrets/blob/main/src/index.ts#L17C63-L17C80
- https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L148-L161

Reply:
added back in!